src/pages/K8s.vue

```diff
@@ -446,12 +453,19 @@ export default {
   display: grid;
   gap: 1em;
   grid-template-areas:
-    "version  port"
-    "kubernetes features";
+    "version port"
+    "kubernetes etc"
+    "traefik flannel";
 
```

Comment:
In reviewing this page, I also noticed that we're missing grid areas for version and port..

```suggestion
.version {
  grid-area: version;
}

.port {
  grid-area: port;
}
```

Finally, we'll need to add the associated classes the corresponding `labeled-input` in the template.